src/Former/Populator.php

```diff
@@ -148,10 +158,16 @@ protected function parseFieldAsArray($field)
 	public function getAttributeFromModel($model, $attribute, $fallback)
 	{
 		if ($model instanceof Model) {
-			return $model->getAttribute($attribute);
+			// Return fallback if attribute is null
+			$value = $model->getAttribute($attribute);
+			return is_null($value) ? $fallback : $value;
```

Comment:
@claar I appreciate you taking the time to look at this. This response is fairly long, so I broke it up into two sections.

**Regarding the scenario you mentioned**

After doing some extra testing, it appears that the scenario you mentioned is actually how the package already works. If a populated value is null, than any value supplied via the `value()` method will be used.

Simple example expanding on what you mentioned (used \App\User since it comes with Laravel)

    $user = new \App\User(['name' => 'from object', 'email' => null]);
    echo Former::open()->populate($user);
    echo Former::text('name')->value('from value');
    echo Former::text('email')->value('from value');
    echo Former::close();

With the above code, the `name` field will have a value of "from object", but the `email` field will have a value of "from value".

This can also be illustrated in the following test:

    // add to tests/Traits/FieldTest.php
    public function testCanPopulateNullOverValueOnFields()
    {
        $this->former->populate(array('foo' => null));
        $static  = $this->former->text('foo')->value('bar')->__toString();
        $matcher = $this->controlGroup('<input id="foo" type="text" name="foo">');

        $this->assertEquals($matcher, $static);
    }

Based on your comment, and the documentation provided at the end of the Features page, it sounds like this is actually an existing issue that should be addressed.

**Regarding this change**

Regarding this particular change, the fallback value referred to is actually the fallback value passed in as the second parameter to the `Populator::get()` method. Looking through the code, it doesn't look like the package itself ever actually uses the fallback value, but provides the ability to use it through the `Former::getValue()` method. Again, this method is never used within the package itself, but is available to the user through the public interface.

Therefore, the only functionality this currently changes would be the following scenario:

    $user = new \App\User(['name' => 'from object', 'email' => null]);
    echo Former::open()->populate($user);
    echo 'Name: '.print_r(Former::getValue('name', 'fallback'), true).PHP_EOL;
    echo 'Email: '.print_r(Former::getValue('email', 'fallback'), true).PHP_EOL;
    echo 'Phone: '.print_r(Former::getValue('phone', 'fallback'), true).PHP_EOL;
    echo Former::close();

    // output before change
    // Name: from object
    // Email: 
    // Phone: 
    
    // output after change
    // Name: from object
    // Email: fallback
    // Phone: fallback

Now, the default for phone works properly. The attribute doesn't exist on the Model, so it should return the fallback. However, this is where your concern comes up: should email continue to return NULL, or should it return the fallback value? To be consistent with plain objects and arrays, it should return NULL.

If you would like it to return NULL, we can update the method as follows:

    if ($model instanceof Model) {
        // Return fallback if attribute is null
        $value = $model->getAttribute($attribute);
        return is_null($value) && !array_key_exists($attribute, $model->toArray()) ? $fallback : $value;
    }

This way, if the value returned from the Model is null, we call `toArray()` on the Model and check if the attribute exists as a key in the array. This tells us if the Model actually had an attribute with the value set to null, or if the attribute was non-existent. Additionally, by using `toArray()`, we are getting all the defined attributes, mutated attributes, as well as the specific relationship, as it would have been loaded by the previous call to `getAttribute()`.

Personally, I feel much more comfortable with this solution. If you would like me to make this change, would you prefer I just modify this PR, or create a new branch and new PR?